docs/configuration.rst

```diff
@@ -48,15 +40,21 @@ below:
      - datadoghq.com
      - Specify which site to use for uploading profiles. Set to
        ``datadoghq.eu`` to use EU site.
-   * - ``DATADOG_TRACE_ENABLED``
+   * - ``DD_TRACE_ENABLED``
      - Boolean
      - True
-     - Enable web framework and library instrumentation. When false, your
-       application code will not generate any traces.
-   * - ``DATADOG_TRACE_DEBUG``
+     - Enable sending of spans to the Agent. Note that instrumentation will still be installed and spans will be
+       generated. Added in ``v0.41.0`` (formerly named ``DATADOG_TRACE_ENABLED``).
```

Comment:
we need to continue to support `DATADOG_TRACE_ENABLED`

Reply:
Sure, but I don't think it's worth documenting it separately if it's deprecated

Reply:
oh yeah, agreed, I just mean the code is no longer using `DATADOG_TRACE_ENABLED`